epub_test.go

```diff
@@ -59,16 +59,16 @@ const (
 	testFontFromFileSource    = "testdata/redacted-script-regular.ttf"
 	testIdentifierTemplate    = `<dc:identifier id="pub-id">%s</dc:identifier>`
 	testImageFromFileFilename = "testfromfile.png"
-	testImageFromFileSource   = "testdata/gophercolor16x16.png"
+	testImageFromFileSource   = "testdata/sample.png"
```

Comment:
As mentioned in the discussion, I don't see the benefit of changing this file. The new one is bigger and the old one wasn't even removed.

Reply:
The "old" one is still being used in some tests. But I will revert the changes to use the same image.

Reply:
ah yes, I see that now. I was only looking in epub_test.go. Yes, I think unless there's a need to use a different image, it's best to keep the old one to keep the repo size smaller.